desktop/core/src/desktop/js/apps/storageBrowser/StorageFilePage/StorageFilePage.tsx

```diff
@@ -31,6 +33,17 @@ const StorageFilePage = ({ fileData }: { fileData: PathAndFileData }): JSX.Eleme
     setIsEditing(true);
   };
 
+  const handleDownload = () => {
+    huePubSub.publish('hue.global.info', { message: t('Downloading your file, Please wait...') });
```

Comment:
I don't understand the need for this complexity? Can't we just create link-button with the url properly set and only add the huePubSub.publish to the onClick? It would also simplify your test.

Reply:
Great feedback. I have fixed it. Thanks.